package.json

```diff
@@ -1,52 +1,32 @@
 {
   "name": "housing-repairs-online-frontend",
-  "version": "0.1.0",
   "private": true,
-  "dependencies": {
-    "@testing-library/jest-dom": "^5.11.4",
-    "@testing-library/react": "^11.1.0",
-    "@testing-library/user-event": "^12.1.10",
-    "govuk-frontend": "^3.13.1",
-    "govuk-react": "^0.9.1",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-router-dom": "^5.3.0",
-    "react-scripts": "4.0.3",
-    "sass": "^1.41.0",
-    "styled-components": "^5.3.1",
-    "web-vitals": "^1.0.1"
-  },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "jest tests/unit",
```

Comment:
Deleting this is causing the action/workflow to fail.
Are unit tests being removed?